.ci/metrics/metrics.py

```diff
@@ -370,6 +370,13 @@ def github_get_metrics(
             started_at = job.started_at
             completed_at = job.completed_at
 
+            if completed_at is None:
+                logging.info(
+                    "Workflow {} is marked completed but has a job without a "
```

Comment:
What are the curly braces for?

Reply:
Was meant to point to an ID through an fstring. Fixed now.